packages/back-end/src/jobs/proxyUpdate.ts

```diff
@@ -141,7 +150,9 @@ export async function queueProxyUpdate(
   if (!CRON_ENABLED) return;
   if (!payloadKeys.length) return;
 
-  const connections = await findSDKConnectionsByOrganization(orgId);
+  const context = await getContextForAgendaJobByOrgId(orgId);
+
+  const connections = await findSDKConnectionsByOrganization(context);
```

Comment:
It looks like the only place calling `queueProxyUpdate` is `triggerWebhookJobs`, which already has a context object.  So I think we can just change the method signature for this function to take context instead of orgId.